src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -384,6 +384,11 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
```

Comment:
Do we know if there is anything in CoreLib that depends on these types? In other words can we successfully remove them from CoreLib?